### DISCUSSION TITLE
deselecting chapters

### DISCUSSION THREAD

**Discussion by exekutive** [mifi/lossless-cut#1821] [Q&A] [answered]
if I have a video with many chapters, how do I deselect them all quickly? I only need to export one. Is clicking each one separately the only way?

**Comment by mifi** [comment]
not sure what you mean by that, care to share a screenshot?

**Comment by exekutive** [comment]
sure
It would be nice to be able to use the shift/command key to select multiple segments , and the have button to toggle them as checked/unchecked.

<img width="1188" alt="Screenshot 2023-12-23 at 15 09 27" src="https://github.com/mifi/lossless-cut/assets/4755215/f446793e-a08f-462d-b920-29bf4827087e">

**Comment by mifi** [comment, marked as answer]
ah you mean segments. right click, then "Deselect all segments"
